app/presenters/finder_presenter.rb

```diff
@@ -4,6 +4,7 @@ class FinderPresenter
 
   delegate :beta_message,
            :document_noun,
+           :context,
```

Comment:
Agreed that it might be too broad but I thought I'd go with what the component uses. @edds?